anime_downloader/extractors/gcloud.py

```diff
@@ -10,22 +10,34 @@
 class Gcloud(BaseExtractor):
     def _get_data(self):
         url = self.url
-        
-        """gcloud uses the same video ID as other sites"""
-        url = url.replace('fembed.com','gcloud.live')
-        url = url.replace('feurl.com','gcloud.live')
-        
-        url = url.replace('gcloud.live/v/','gcloud.live/api/source/')
-        if url.find('#') != -1:url = url[:url.find('#')] 
-        url = (url[-url[::-1].find('/'):])
-        data = helpers.post(f'https://gcloud.live/api/source/{url}').json()['data']
 
+        # TODO: Maybe put it in Option ?
+        withPreserveServerPreference = True
+        
+        if not withPreserveServerPreference:  
+            """gcloud uses the same video ID as other sites"""
+            url = url.replace('fembed.com','gcloud.live')
+            url = url.replace('feurl.com','gcloud.live')
+            
+            url = url.replace('gcloud.live/v/','gcloud.live/api/source/')
+            if url.find('#') != -1:url = url[:url.find('#')] 
```

Comment:
This gcloud is outdated and based on a pretty horrible old extractor, to prevent conflicts and all that I'd suggest building from the newer one